ddprof-lib/src/main/cpp/flightRecorder.cpp

```diff
@@ -143,13 +143,8 @@ void Lookup::fillJavaMethodInfo(MethodInfo* mi, jmethodID method, bool first_tim
 
         if (jvmti->GetClassModifiers(method_class, &class_modifiers) == 0 && jvmti->GetMethodModifiers(method, &modifiers) == 0) {
             // class constants are written without the modifiers info
-            // in order to be able to identify 'hidden' frames the "SYNTHETIC" and "BRIDGE" modifiers will be propagated to methods
-            if (class_modifiers & 0x1000) {
-                modifiers |= 0x1000;
-            }
-            if (class_modifiers & 0x0040) {
-                modifiers |= 0x0040;
-            }
+            // in order to be able to identify 'hidden' frames the "SYNTHETIC" (0x1000) and "BRIDGE" (0x0040) modifiers will be propagated to methods
+            modifiers |= class_modifiers & (0x1000 | 0x0040);
```

Comment:
(bikeshed) maybe pull these masks into constants and replace with `modifiers |= class_modifiers & (SYNTHETIC | BRIDGE);` and remove the comment